lib/winston/transports.js

```diff
@@ -12,23 +12,30 @@ var fs = require('fs'),
 
 var transports = exports;
 
-//
-// Setup all transports as lazy-loaded getters.
-//
-fs.readdirSync(path.join(__dirname, 'transports')).forEach(function (file) {
-  var transport = file.replace('.js', ''),
-      name  = common.capitalize(transport);
-
-  if (transport === 'transport') {
-    return;
-  }
-  else if (~transport.indexOf('-')) {
-    name = transport.split('-').map(function (part) {
-      return common.capitalize(part);
-    }).join('');
-  }
-
-  transports.__defineGetter__(name, function () {
-    return require('./transports/' + transport)[name];
-  });
-});
\ No newline at end of file
+transports.__defineGetter__('CONSOLE', function () {
```

Comment:
Casing is wrong on all of these. Would be:

``` js
transports.__defineGetter__('Console', function () {
```

Also you should use `Object.defineProperty` not `__defineGetter__`